packages/file-type-icons/src/initializeFileTypeIcons.tsx

```diff
@@ -8,43 +8,69 @@ const SVG_SUFFIX = '_svg';
 const DEFAULT_BASE_URL = 'https://spoprod-a.akamaihd.net/files/fabric-cdn-prod_20200921.001/assets/item-types/';
 const ICON_SIZES: number[] = [16, 20, 24, 32, 40, 48, 64, 96];
 
+const _fileTypeIconsAsString: { [iconName: string]: string } = {};
+
 export function initializeFileTypeIcons(baseUrl: string = DEFAULT_BASE_URL, options?: Partial<IIconOptions>): void {
   ICON_SIZES.forEach((size: number) => {
     _initializeIcons(baseUrl, size, options);
   });
 }
 
+export function getFileTypeIconsAsHTMLStrings(iconName: string): string | undefined {
+  if (_fileTypeIconsAsString.hasOwnProperty(iconName)) {
+    return _fileTypeIconsAsString[iconName];
+  }
+}
+
 function _initializeIcons(baseUrl: string, size: number, options?: Partial<IIconOptions>): void {
   const iconTypes: string[] = Object.keys(FileTypeIconMap);
   const fileTypeIcons: { [key: string]: JSX.Element } = {};
 
   iconTypes.forEach((type: string) => {
     const baseUrlSizeType = baseUrl + size + '/' + type;
-    fileTypeIcons[type + size + PNG_SUFFIX] = <img src={baseUrlSizeType + '.png'} alt="" />;
-    fileTypeIcons[type + size + SVG_SUFFIX] = <img src={baseUrlSizeType + '.svg'} alt="" />;
+    let src: string;
+    let iconName: string;
+
+    src = `${baseUrlSizeType}.png`;
+    iconName = type + size + PNG_SUFFIX;
+    fileTypeIcons[iconName] = <img src={src} alt="" />;
```

Comment:
minor question. Does this alt do something? Looks like it doesn't play any role here.

Reply:
May be wrong, but I believe this is due to accessibility scanners which will warn when you have an image without considering the alt tag (even providing a blank one is considered "accessible")...

Reply:
Might not be a bad idea to populate this alt text with `type + " icon"` ~ the type is not always super descriptive but we try our best to make it so.

Reply:
That is a good suggestion! Will take that up as a separate PR. Thanks!